docs/src/components/Layout/Footer.tsx

```diff
@@ -19,14 +19,22 @@ export const Footer = () => {
         <Button
           as={Link}
           variation="link"
+          // @ts-ignore // IGNORE
           href={GITHUB_REPO}
           isExternal
           gap="xs"
         >
           <GithubIcon ariaLabel="" />
           Contribute on GitHub
         </Button>
-        <Button as={Link} variation="link" href={DISCORD} isExternal gap="xs">
+        <Button
+          as={Link}
+          variation="link"
+          // @ts-ignore // IGNORE
+          href={DISCORD}
```

Comment:
@dbanksdesign I think we were considering adding Button styling for the Link component to solve this issue?

Reply:
Correct, I can get a PR for that